athena/active.py

```diff
@@ -318,3 +324,26 @@ def _rotate_x(self, reduced_inputs, inactive_inputs):
         inputs = np.dot(YZ, self.evects.T).reshape((N * NY, m))
         indices = np.kron(np.arange(NY), np.ones(N)).reshape((N * NY, 1))
         return inputs, indices
+
+    def _frequent_directions(self, gradients):
+        """
+        Function that perform the frequent directions algorithm for 
+        matrix sketching. For more details about the method, see
+        "Frequent directions: Simple and deterministic matrix
+        sketching." Ghashami, Mina, et al.
+        SIAM Journal on Computing 45.5 (2016): 1762-1792.
+       
+        :param iterable gradients: generator for spatial gradients
+        :return numpy.ndarray sigma and v: matrix of singular values
```

Comment:
please rephrase. For example 'projection matrix' is more correct. Also the type format is not correct: refer to the documentation of `_build_decompose_cov_matrix` in subspace.py for example.